scripts/build_base_energy_totals.py

```diff
@@ -26,20 +31,34 @@ def calc_sector(sector):
         # print(country, sector)
         df_co = df_yr[df_yr.country == country]
 
-        if sector != "navigation":
+        if sector == "navigation":
```

Comment:
Great @hazemakhalek.

I'm giving here my five cents if useful.
This if/elif case seems quite heavy but operations sounds quite similar across them:
1. Select a column of the dataframe, it being "Commodity - Transaction" for navigation and the else case; Transaction for the other sectors
2. Some data cleaning lower(), replace - into a space
3. Finally select a subset of values

At first glance, it seems that some operations are repeated and it should be possible to condensate them; what do you think?

Reply:
Added to issue: https://github.com/pypsa-meets-earth/pypsa-earth-sec/issues/358